wallet-new/src/Cardano/Wallet/Kernel.hs

```diff
@@ -18,6 +18,7 @@ module Cardano.Wallet.Kernel (
   , init
   , walletLogMessage
   , walletPassive
+  , walletKeystore
```

Comment:
Check if this is really needed.

Reply:
EDIT: Unfortunately it is, because we need it in the `createAddress` function.

Reply:
Maybe at some point we should rename `.Kernel` to `.Kernel.Internal` and export only the truly public API from `.Kernel`. Doesn't need to be now though obviously.